jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JsonEnumDefaultValueResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.github.victools.jsonschema.module.jackson;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.github.victools.jsonschema.generator.TypeScope;
```

Comment:
As per below suggestion, declaring the more specific `FieldScope` instead of `TypeScope` as method parameter type.
```suggestion
import com.github.victools.jsonschema.generator.FieldScope;
```